static/js/components/CommentList.jsx

```diff
@@ -150,19 +147,11 @@ const CommentList = ({ isCandidate }) => {
         )}
       </div>
       <br />
-      {!isCandidate && permissions.indexOf("Comment") >= 0 && (
+      {permissions.indexOf("Comment") >= 0 && (
```

Comment:
How does this PR affect comment viewing permissions on a source?

Reply:
No effect. You still need to be a member of the groups a comment is shared with in order to see a comment, same logic as before.